### PR TITLE
«api and books»

### PR DESCRIPTION
**Pull request by ocean4211**

### CODE REVIEW

File: api.php [hunk 1]
```diff
@@ -1,16 +1,12 @@
 <?php
 require_once('books.php');
```
Comment (nitro5002) [inline api.php]: Filename should be exactly the same as class name (Books.php)

File: books.php [hunk 2]
```diff
@@ -1,46 +1,36 @@
...
+    {
+        $query = "SELECT * FROM book";
+        $resQuery = $this->link->query($query);
+        echo (json_encode($resQuery->fetchAll(PDO::FETCH_ASSOC)));
+    }
+    public function booksId($id)
+    {
+        $query = "SELECT * FROM book WHERE id = $id";
+        $resQuery = $this->link->query($query);
+        echo json_encode($resQuery->fetch(PDO::FETCH_ASSOC));
```
Comment (nitro5002) [inline books.php:28]: You can use fetchColumn to obtain one value.
https://www.php.net/manual/en/pdostatement.fetchcolumn.php

File: books.php [hunk 3]
```diff
@@ -1,46 +1,36 @@
...
+    }
+    public function count()
+    {
+        $query = "SELECT * FROM book";
```
Comment (nitro5002) [inline books.php]: SQL provides function COUNT() to count amount of rows in table.
https://dev.mysql.com/doc/refman/5.7/en/counting-rows.html
https://www.w3schools.com/sql/func_mysql_count.asp
Then you again can you fetchColumn like in previous comment.

**Review by nitro5002 — CHANGES_REQUESTED** [review]
Please respond to comments and/or update code.

File: books.php (hunk 2 above)
Comment (ocean4211) [reply books.php:28]: fetchColumn gives only one first value of row, but I want all values of needed row (description chosen books by id)

**Comment by nitro5002** [comment]
Approved. Please merge.

**Review by nitro5002 — APPROVED** [review]

**Pull request merged**